File: app/src/main/java/com/ianfield/bodyscoring/activity/AboutActivity.java
package com.ianfield.bodyscoring.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.webkit.WebView;

import com.ianfield.bodyscoring.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class AboutActivity extends AppCompatActivity {

    @BindView(R.id.webview)
    WebView webView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        ButterKnife.bind(this);
        webView.loadUrl("file:///android_asset/about.html");
    }
}

File: app/src/main/java/com/ianfield/bodyscoring/activity/AboutActivity.kt
package com.ianfield.bodyscoring.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.ianfield.bodyscoring.R


import kotlinx.android.synthetic.main.activity_about.*

class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        webview.loadUrl("file:///android_asset/about.html")
    }
}
